core_backend/tests/api/test_manage_content.py

```diff
@@ -44,6 +51,78 @@ def existing_content_id(
     )
 
 
+@pytest.fixture(scope="class")
+def temp_user(client: TestClient, db_session: Session) -> Generator[str, None, None]:
```

Comment:
Would be good to create two users (using params) with different `content_quota`.

Reply:
Done :)